source/documentors/references/doc_checklist.rst

```diff
@@ -102,4 +120,52 @@ In the pull request, click **Details** to see the documentation with your change
 
 If there is an error when building the documentation, the pull request will indicate that checks failed, and the **Details** link will take you to the error message. If you are able to fix the issue indicated by the error message, do so; if not, contact the Open edX team for assistance.
 
+Migrating Docs Checklist
+************************
+
+Some checklist items are specific to migrating 2U/Edx.org legacy docs into Open edX® Docs. During migration, documentors are using this `Open EdX Doc Migration Tracking`_ sheet.
+
+Remove or modify references that are specific to 2U/EdX.org
+========================================================================
+
+When migrating from legacy documenation from 2U/EdX.org, remove references that are applicable only to the 2U or EdX.org users.
+
+Modify references that may have come from 2U/EdX.org but are also applicable to Open edX® LMS users.
```

Comment:
I'm not sure why you mention LMS specifically here. If that's important, please ignore my comment.
```suggestion
Modify references that may have come from 2U/EdX.org but are also applicable to Open edX® users.
```

Reply:
I have been told, at times, to use Open edX as an adjective (so always have a noun like platform or LMS or community after it). So leaving this one as is.